backend/routes/donation.js

```diff
@@ -88,16 +89,21 @@ router.post("/", (req, res) => {
                         .send(
                           "Error has occured during the post of the new donation !"
                         );
-                    return res
-                      .status(201)
-                      .send("Donation posted successfully !");
+                    res.status(201).send("Donation posted successfully !");
+                    sendMail({
+                      from: "Team.Meex@meex.com",
+                      to: email,
+                      replyTo: "Team.Meex@meex.com",
+                      subject: "Thank you for your donation !",
+                      html: `Thank you! You just donated ${donation_value}€ and by doing this you participated to have a better world !`
```

Comment:
(While correct, this message looks strange in the case of a 0€ donation, doesn't it ? To make it better, you could create another message and switch it in this specific case)